_posts/2016-05-26-Rust-1.9.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "Announcing Rust 1.9"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.9. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.9][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.9][notes] on GitHub.
+About 1400 patches were landed in this release.
```

Comment:
Sorry, this number isn't accurate yet; I held off on all the contributor stuff for now.